### PR TITLE
[New ruleset] useremarkable.com

### PR DESCRIPTION
**Pull request by Riotism**

### CODE REVIEW

File: src/chrome/content/rules/useremarkable.com.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<ruleset name="Remarkable">
```
Comment (J0WI) [inline src/chrome/content/rules/useremarkable.com.xml]: Please also add `useremarkable.com` (without `www`).

**Comment by jeremyn** [comment]
@Riotism Are you still interested in working on this pull request?

**Comment by Riotism** [comment]
Yes. Is there anything else I need to fix?

File: src/chrome/content/rules/useremarkable.com.xml (hunk 1 above)
Comment (Riotism) [reply src/chrome/content/rules/useremarkable.com.xml]: Done.

**Comment by jeremyn** [comment]
Please:
- [ ] Change the `ruleset` `name` to `UseRemarkable.com`
- [ ] Change the filename to `UseRemarkable.com.xml`
- [ ] Add a `test` for `oup` like this
```xml
	<target host="oup.useremarkable.com" />
		<test url="http://oup.useremarkable.com/production/images/uploads/3677/original/business_letter_template_new_colleague.pdf" />
```
- [ ] (Optional) Add a `test` for `oupm` if you can find one, though note that `test`s should not be personalized in any way (that is, not real content for a person, and ideally not even resembling real content for a person). If you can't find a suitable URL, that's fine, just say so.

**Comment by jeremyn** [comment]
Your changes look okay. Can you please confirm that you don't plan to add a `test` for `oupm`? In other words, please confirm that you are done making changes.

**Comment by Riotism** [comment]
I am done making changes. I can't find a suitable URL for `oupm`

**Comment by jeremyn** [comment]
Okay thanks. One last thing, can you please remove the extra tab before `rule`? I'm sorry I didn't catch that before.

**Pull request merged**